gridy/Sources/Presentations/PlanBoard/PlanBoardView.swift

```diff
@@ -110,24 +176,257 @@ extension PlanBoardView {
+                    
+                    let gridWidth = (geometry.size.width - viewStore.columnStroke * CGFloat(viewStore.map.count - 1)) / CGFloat(viewStore.map.count)
+                    /// hover 되었을 때
+                    if viewStore.hoveredItem == PlanBoardAreaName.listArea.rawValue {
+                        if let hoveredRow = viewStore.listAreaHoveredCellRow, let hoveredCol = viewStore.listAreaHoveredCellCol {
+                            Rectangle()
+                                .fill(Color.itemHovered)
+                                .frame(
+                                    width: gridWidth,
+                                    height: viewStore.lineAreaGridHeight - viewStore.rowStroke
+                                )
+                                .position(
+                                    x: gridWidth / 2 + (gridWidth + viewStore.columnStroke) * CGFloat(hoveredCol),
+                                    y: CGFloat(Double(hoveredRow) + 0.5) * viewStore.lineAreaGridHeight - viewStore.rowStroke / 2
+                                )
+                                .highPriorityGesture(TapGesture(count: 1).onEnded({
+                                    // TODO: - click 시 선택되어 보이는 사각형, drag와 함께 작업
+                                    viewStore.send(.lineIndexAreaClicked(false))
+                                }))
+                                .simultaneousGesture(TapGesture(count: 2).onEnded({
+                                    listItemFocused = true
+                                    viewStore.send(.listItemDoubleClicked(.listItem, false))
+                                    viewStore.send(.listItemDoubleClicked(.emptyListItem, true))
+                                    viewStore.send(.setHoveredLocation(.listArea, false, nil))
+                                }))
+                                .contextMenu {
+                                    Button("Delete this Plan") {
+                                        /// Dummy ListItem View에도 일관성을 주기 위한 버튼으로 아무 액션도 수행하지 않음
+                                    }
+                                }
+                                .opacity((viewStore.selectedEmptyRow == hoveredRow)&&(viewStore.selectedEmptyColumn == hoveredCol) ? 0 : 1)
+                        }
+                    }
+                    
+                    /// double click 되었을 때
+                    if let columnOffset = viewStore.selectedEmptyColumn, let rowOffset = viewStore.selectedEmptyRow {
+                        Rectangle()
+                            .fill(Color.clear)
+                            .overlay(
+                                TextField("editing",
+                                          text: viewStore.binding(
+                                            get: \.keyword,
+                                            send: { .keywordChanged($0) }
+                                          ))
+                                .focused($listItemFocused)
```

Comment:
이 부분 때문에 BindingState로 변경되었군요~ 저도 궁금했던 건데 내친 김에 찾아보았습니다
state를 binding 가능한 형태로 만들 수 있는 방법이 여러가지가 있는데, 이 코드에서는 명시적으로 binding 액션을 선언해주고 있어 텍스트가 바뀔때마다 .keywordChanged가 실행되어 값이 업데이트 될 수 있습니다!
그리고 @BindingState를 사용한다면 이 코드에서는 아래와 같이 코드를 작성해야 합니다
```suggestion
                                TextField("editing",
                                          text: viewStore.$keyword
                                          ))
                                .focused($listItemFocused)
```
문제가 된다기보다는 중복으로 사용하고 있는 상태입니다 !-! 개행까지 수정한다면 아래처럼 쓸 수 있겠네용~
```suggestion
                                TextField(
                                          "editing",
                                          text: viewStore.$keyword
                                          )
                                 )
                                .focused($listItemFocused)
```

https://pointfreeco.github.io/swift-composable-architecture/1.0.0/documentation/composablearchitecture/bindings/
공식문서를 보면 더 이해가 쉬울 것 같습니다 참고 🫰

Reply:
개행완료~!